### PR TITLE
Add Rubocop

### PR DESCRIPTION
**Pull request by benbalter**
Since I'm on a recent Rubocop kick (excluding a handful of tests that we're currently failing).

### CODE REVIEW

File: bin/github-pages [hunk 1]
```diff
@@ -1,54 +1,54 @@
 #!/usr/bin/env ruby
 
 require 'mercenary'
-require_relative "../lib/github-pages"
+require_relative '../lib/github-pages'
```
Comment (parkr) [inline bin/github-pages]: I thought it was double quotes everywhere. Let's get a rubocop file going for [GitHub's Ruby Styleguide](https://github.com/styleguide/ruby)

Comment (benbalter) [inline bin/github-pages]: TIL'd. Makes sense.

**Comment by parkr** [comment]
@benbalter you might want to do a new pr - looks like there are some conflicts here

**Comment by benbalter** [comment]
> looks like there are some conflicts here

Fixed. Assuming tests passed, I believe this should be good to go (I used your config)

**Comment by parkr** [comment]
:shipit:

**Pull request merged**